bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/dtree/Root.scala

```diff
@@ -8,7 +8,7 @@ class Root(sem:Semester) extends SingleSelectNode[Semester, Instrument, Any](sem
   var title = "Select Instrument"
   var description = s"The following instruments are available for semester ${sem.year}${sem.half}. See the Gemini website for information on instrument capabilities and configuration options."
 
-  def choices = List(Alopeke, Flamingos2, GmosNorth, GmosSouth, Gnirs, Gpi, Graces, Gsaoi, Nifs, Niri, Visitor, Zorro)
+  def choices = List(Alopeke, Flamingos2, GmosNorth, GmosSouth, Gnirs, Gpi, Graces, Gsaoi, Igrins, Nifs, Niri, Visitor, Zorro)
 
   def apply(i:Instrument) = i match {
```

Comment:
Here the type is maybe something like
```scala
Either[SelectNode[Instrument, _, _, GeminiBlueprintBase], GeminiBlueprintBase]
```

Reply:
Trying both `SelectNode` and `Node` doesn't seem to work, either. I get, for each instrument, something akin to:
```
Error:(14, 41) type mismatch;
 found   : edu.gemini.model.p1.dtree.inst.Alopeke.ModeNode
 required: edu.gemini.model.p1.dtree.SelectNode[edu.gemini.model.p1.immutable.Instrument, _, _, edu.gemini.model.p1.immutable.GeminiBlueprintBase]
    case Alopeke    => Left(inst.Alopeke())
```

I'll take another look at it tomorrow.

Reply:
Note this is for:
```
  def apply(i:Instrument): Either[SelectNode[Instrument, _, _, GeminiBlueprintBase], GeminiBlueprintBase] = i match {
     case Alopeke    => Left(inst.Alopeke())
     ...
```

Reply:
It looks like `Alopeke.ModeNode` is a `SingleSelectNode` where the instrument parameter is just `Unit` and not really an `Instrument`.  Don't worry I don't think you should spend too much time on this since it has always been this way.